docs/03-tutorials/00-api-exposure/apix-02-create-workload.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Create a workload
+---
+
+The tutorial comes with a sample HttpBin service deployment and a sample Function.
+
+The tutorial may be a follow-up to the [Use a custom domain to expose a workload](./apix-01-own-domain.md) tutorial.
```

Comment:
```suggestion
It may be a follow-up to the [Use a custom domain to expose a workload](./apix-01-own-domain.md) tutorial.
```

Reply:
Or: `You can use it as a follow up...` - to avoid the repetition of `this tutorial`.